toolkit/popup/src/main/java/com/arcgismaps/toolkit/popup/Popup.kt

```diff
@@ -107,9 +114,22 @@ private val attachments: MutableList<PopupAttachment> = mutableListOf()
 @Composable
 private fun Popup(popupState: PopupState, modifier: Modifier = Modifier) {
     val popup = popupState.popup
+    val dynamicEntity = (popup.geoElement as? DynamicEntity)
     var evaluated by rememberSaveable(popup) { mutableStateOf(false) }
     var fetched by rememberSaveable(popup) { mutableStateOf(false) }
-
+    var refreshed by rememberSaveable(dynamicEntity) { mutableStateOf(true) }
+    if (dynamicEntity != null) {
+        LaunchedEffect(popup) {
+            dynamicEntity.dynamicEntityChangedEvent.collect { info ->
```

Comment:
Unused param here, could be removed. 

```suggestion
            dynamicEntity.dynamicEntityChangedEvent.collect { 
```